package.nls.zh.json

```
@@ -0,0 +1,29 @@
+{
+  "vscodePets.spawnPet": "产卵宠物",
+  "vscodePets.petPanel": "宠物面板",
+  "vscodePets.selectPet": "选择宠物",
+  "vscodePets.selectColor": "选择颜色",
+  "vscodePets.selectPetToRemove": "选择宠物去除。",
+  "vscodePets.leaveBlank": "留空白名称",
+  "vscodePets.nameYourPet": "命名您的宠物",
+  "vscodePets.cancelledSpawningPet": "取消产卵宠物",
+  "vscodePets.spawnPetWithClosedPlayground": "已经创建了一个宠物游乐场。 现在，您可以使用\"Spawn Additional Pet\"命令添加更多宠物。",
+  "vscodePets.removePetsWithClosedPlayground": "已经创建了一个宠物游乐场。 现在，您可以使用\"Remove All Pets\"命令去除所有宠物。",
+  "vscodePets.black": "黑色的",
+  "vscodePets.brown": "棕色的",
+  "vscodePets.green": "绿色的",
+  "vscodePets.yellow": "黄色",
+  "vscodePets.gray": "灰色的",
+  "vscodePets.red": "红色的",
+  "vscodePets.white": "白色的",
+  "vscodePets.cat": "猫",
+  "vscodePets.clippy": "笨拙",
+  "vscodePets.cockatiel": "小鹦鹉",
+  "vscodePets.crab": "螃蟹",
+  "vscodePets.dog": "狗",
+  "vscodePets.rocky": "岩石",
+  "vscodePets.rubber-duck": "橡皮鸭",
+  "vscodePets.snake": "蛇",
+  "vscodePets.totoro": "托托罗",
+  "vscodePets.zappy": "活泼的"

```

```suggestion
  "vscodePets.zappy": "zappy"

```

It is a name that we should not translate